src/amo/components/HomeHeroBanner/index.js

```diff
@@ -206,26 +234,51 @@ export class HomeHeroBannerBase extends React.Component<Props> {
     ];
 
     return heroes.map((hero) => {
+      const { title, url } = hero;
       return (
-        <HeroSection key={hero.url} linkTo={hero.url}>
-          <h3>{hero.title}</h3>
+        <HeroSection
+          key={url}
+          linkTo={url}
+          experimentTitle={title}
```

Comment:
note: I added this here so it would be available for the tests. 
Also I named it so we know to remove this when the test is done.

Reply:
mmh that does not look good to me because tests should not change our production code.